pro/cypress/e2e/features/counter.feature

```diff
@@ -0,0 +1,44 @@
+@P0
+Feature: Counter (Guichet) feature
+
+  Background:
+    Given I am logged in with new interface
```

Comment:
Alors je suis peut être pas passé sur la revue qui a apporté ces steps mais on pourra améliorer (pas forcément cette PR pour les steps communes) pour avoir de l'anglais plus correct : 

`Given I am logged in with *the* new interface`

`And I go to *the* "Guichet" page`

`It should display *the* identity check message`

`Then *the* identity check message is displayed`

`And I can click and open *the* "Modalités de retrait et CGU" page`

Reply:
OK corrigé